packages/sx.js/src/highlightConstants.ts

```diff
@@ -15,6 +27,19 @@ export const ALIASES_CONFIG = {
   }
 };
 
+export const STARKNET_ALIASES_CONFIG = {
+  address: '0x0000000000000000000000000000000000000003',
+  types: {
+    setAlias: {
+      StarknetDomain: STARKNET_DOMAIN_TYPE,
```

Comment:
Is it used there? I see that we are stripping it in the agent, but I can't find a place where it's actually used on `STARKNET_ALIASES_CONFIG`.